cocos/2d/CCFontFNT.cpp

```diff
@@ -393,9 +394,10 @@ void BMFontConfiguration::parseImageFileName(const char* line, const std::string
     int pageId;
     sscanf(line, "page id=%d", &pageId);
     CCASSERT(pageId == 0, "LabelBMFont file could not be found");
+
     // file 
     char fileName[255];
-    sscanf(strchr(line,'"') + 1, "%[^\"]", fileName);
+    sscanf(strstr(line, "file=\"") + 6, "%[^\"]", fileName);
```

Comment:
Will it break compatibility?

Reply:
Before it just checked for a `"`, but it's not a good way to handle it.  It's better to explicitly set what you're searching for in the string.

For example, it looks at this line in the FNT file:

```
page id=0 file="arimo_bold30_0.png"
```

The current code searches just for a `"`, which works fine.  The modified code just searches for the whole `file="` instead, to make it clearer what the purpose is.  I can revert it if you prefer, since it really makes no difference.

Reply:
Got it, thanks. Let's keep it.